conftest.py

```diff
@@ -0,0 +1,114 @@
+
+
+@pytest.fixture
+def daycare2_message_to_dogowner():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=2),
+                          text='Hello this new chat between daycare2 and dogowner')
+
+
+@pytest.fixture
+def daycare3_message_to_dogowner():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=3),
+                          text='new chat between daycare3 and dogowner')
+
+
+@pytest.fixture
+def create_dog_owner_user():
+    return DogOwner.create(email='testuser@gmail.com',
+                           username='testuser01',
+                           password='testpassowrd',
+                           dog_name='kliford',
+                           first_name='NEW',
+                           last_name='USER',
+                           phone_number=1234567890,
+                           dog_race='lavrador',
+                           dog_picture_url="https://www.akc.org/wp-content/uploads/2019/06/Bohemian-Shepherd.1.jpg",
+                           dog_age=10,
+                           dog_weight=6,
+                           dog_gender='M'
+                           )
+
+
+@pytest.fixture
+def create_daycare_user():
+    return DayCare.create(email="test@gmail.com", username="testuser01", password="pass",
+                          name="Puppies", description="This is the first daycare test",
+                          price_per_day=10,
+                          capacity=50, area="Merkaz", city="Tel-Aviv", address="The best street 5")
+
+
+@pytest.fixture
+def create_image1(create_daycare_user):
+    return Image.create(url="../../static/images/daycare_image_test_01.jpeg", daycare_id=create_daycare_user)
+
+
+@pytest.fixture
+def create_image2(create_daycare_user):
+    return Image.create(url="../../static/images/daycare_image_test_02.jpeg", daycare_id=create_daycare_user)
+
+
+@pytest.fixture
+def daycare():
+    return DayCare.create(email="daycare_review@gmail.com", username="daycare_review", password="pass",
+                          name="reviewDayCare", description="This is test daycare for review", price_per_day=10,
+                          capacity=50, area="Merkaz", city="Tel-Aviv", address="The best review")
+
+
+@pytest.fixture
+def dogowner():
+    return DogOwner.create(email="dogowner_review@gmail.com", username='dogowner_review', password='pass',
+                           dog_name='dogReview', first_name='firstNameReview', last_name='lastNameReview',
+                           phone_number=1234567890, dog_race='review',
+                           dog_picture_url='https://www.google.com/review.jpg', dog_age=1, dog_weight=1, dog_gender='M')
+
+
+@pytest.fixture
+def review_data():
```

Comment:
I suggest creating such fixture for DogOwner and DayCare so we won't use hard-coded values in the tests, See comments in the tests